source/2019-12-19-countdown-to-the-new-year-ember-template-lint.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Countdown to The New Year - Ember Template Lint
+author: Melanie Sumner
+tags: 2019, Accessibility, Addons, DecEmber
+alias: 2019-12-19-countdown-to-the-new-year-ember-template-lint.md
+responsive: true
+---
+
+**This is the 19th in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 19
+
+Today we are going to look at an addon that, while it is built into Ember apps by default, has had enough updates lately to deserve a closer look- [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint).
```

Comment:
```suggestion
Today we are going to look at an addon that is part of every new Ember app. [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) has many new features added recently, and is worth a close look!
```

Rewording because the sentence seemed too complex